nix/marlowe-cardano/deploy/operables.nix

```diff
@@ -121,6 +126,35 @@ let
 
   mkOperableWithProbes = args: mkOperable (args // probes);
 
+  mkMarloweAppScript = script: ''
+    #################
+    # REQUIRED VARS #
+    #################
+    # MARLOWE_RT_HOST: Host name of the marlowe runtime server.
+
+    [ -z "''${MARLOWE_RT_HOST:-}" ] && echo "MARLOWE_RT_HOST env var must be set -- aborting" && exit 1
+
+    #################
+    # OPTIONAL VARS #
+    #################
+    # MARLOWE_RT_PORT: Port number of the marlowe runtime server.
+    # TIMEOUT_SECONDS: Timeout in seconds for transaction confirmation.
+    # BUILD_SECONDS: Wait specified seconds before transaction construction.
+    # CONFIRM_SECONDS: Wait specified seconds after transaction confirmation.
+    # RETRY_SECONDS: Wait specified seconds after a failed transaction before trying again.
+    # RETRY_LIMIT: Maximum number of attempts for trying a failed transaction again.
+
```

Comment:
Do we need to add the following because an `export` is required for the app to see the variable?
```bash
export MARLOWE_RT_HOST="''${MARLOWE_RT_HOST}"
```

Reply:
That should not be necessary

Reply:
Also, the connection is successful, but something is going wrong over the wire

Reply:
You would see a different error if the connection failed, not a `DeserializeError`